content/micropython/videoInfo.md

```diff
@@ -6,25 +6,25 @@ info:
       icon: IconBoard,
       title: Board Installation,
       description: Create smart dashboards to control connected devices using few coding,
```

Comment:
The descriptions are also really weird.

Reply:
Very fair @jacobhylen I have changed them.